tests/safety/test_gm.py

```diff
@@ -18,7 +18,7 @@ class Buttons:
   CANCEL = 6
 
 
-class TestGmSafety(common.PandaSafetyTest, common.DriverTorqueSteeringSafetyTest):
+class TestGmSafetyBase(common.PandaSafetyTest, common.DriverTorqueSteeringSafetyTest):
   TX_MSGS = [[384, 0], [1033, 0], [1034, 0], [715, 0], [880, 0],  # pt bus
```

Comment:
Self note:

- One base test class (only subclassed from)
- A class for ASCM with op long
- A class for GM CAM (only stock long, we can subclass that to add op long tests)